chaos/net-rpc-client.go

```diff
@@ -75,10 +76,7 @@ func (rpcClient *ReconnectRPCClient) dialRPCClient() (*rpc.Client, error) {
 }
 
 // Call makes a RPC call to the remote endpoint using the default codec, namely encoding/gob.
-func (rpcClient *ReconnectRPCClient) Call(serviceMethod string, args interface {
```

Comment:
With this removal where does the Token and Timestamp will be validated? .  Is the purpose to remove that to not enforce authentication behavior?